view_conversations/conv-viewer-webapp/src/app/conversation/conversation.component.html

```diff
@@ -2,7 +2,8 @@
   <table class="action" [ngClass]="{ 'finalcomment': isHighlighted}">
     <tr>
       <td class="whenandwho">
-        <div class="author">{{rootComment.user_text}}</div>
+        <div class="author" *ngIf="rootComment.user_text">{{rootComment.user_text}}</div>
+        <div class="author" *ngIf="!rootComment.user_text">Anonymous</div>
```

Comment:
Maybe add some styling to distinguish the user_text = "Anonymous" vs user_text is not specified?